test/common/tmpdir.js

```diff
@@ -5,17 +5,27 @@ const fs = require('fs');
 const path = require('path');
 const { pathToFileURL } = require('url');
 const { isMainThread } = require('worker_threads');
+const isUnixLike = process.platform !== 'win32';
+const { escapePOSIXShell } = require('./index.js');
```

Comment:
Looks like you need to lazy load to workaround the circular dependency

Reply:
Thanks! Updating PR now